backend-elasticsearch/src/main/java/org/hibernate/search/v6poc/backend/elasticsearch/document/model/impl/ElasticsearchIndexModel.java

```diff
@@ -18,11 +18,22 @@
 
 	private final String indexName;
 	private final TypeMapping mapping;
+	private final Map<String, ElasticsearchObjectNodeModel> objectFieldModels = new HashMap<>();
 	private final Map<String, ElasticsearchFieldModel> fieldModels = new HashMap<>();
 
 	public ElasticsearchIndexModel(String indexName, ElasticsearchRootIndexSchemaCollectorImpl collector) {
 		this.indexName = indexName;
-		this.mapping = collector.contribute( fieldModels::put );
+		this.mapping = collector.contribute( new ElasticsearchIndexModelCollector() {
```

Comment:
Isn't it an ElasticsearchIndexSchemaCollector?

I thought we decided that Model = Schema + other things. As we add elements to the root IndexSchemaCollector, I would have expected Schema in there.

Reply:
Ok, will do. I expect we'll have to rename it to *Model* ultimately, but then we'll have to rename some SPIs too, so let's be consistent in the meantime.